tests/contrib/hooks/test_gcs_hook.py

```diff
@@ -558,7 +558,7 @@ def test_download_to_file(self, mock_service):
                                           object_name=test_object,
                                           filename=test_file)
 
-        self.assertEqual(response, test_object_bytes)
```

Comment:
Does this test now also leave `test_file` file hanging around? (I think it does.) We should tidy that up with a `try`/`finally`.